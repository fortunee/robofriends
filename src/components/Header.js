import React from 'react';

const Header = () => <h1 className="f1">Robo friends</h1>;

export default Header;
