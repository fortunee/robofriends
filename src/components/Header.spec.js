import { shallow } from 'enzyme';
import React from 'react';

import Header from './Header';

it('Should render a Header component', () => {
  expect(shallow(<Header />).debug()).toMatchSnapshot();
});
