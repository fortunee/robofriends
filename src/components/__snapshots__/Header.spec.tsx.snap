// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render a Header component 1`] = `
"<h1 className=\\"f1\\">
  Robo friends
</h1>"
`;
